tool_quality_checks/quality_checks.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+""" TODO Docstring. """
+import os
+import argparse
+import logging
+
+from configparser import RawConfigParser
+from db import ThreediDatabase
```

Comment:
Dit werkt alleen op python2: relatieve import. `from .db import ...` werkt waarschijnlijk wel.

Reply:
ImportError while importing test module 'D:\Documents\GitHub\oas-dgl-datachecker\tool_quality_checks\test\test_quality_checks.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tool_quality_checks\test\test_quality_checks.py:8: in <module>
    import quality_checks
tool_quality_checks\quality_checks.py:8: in <module>
    from .db import ThreediDatabase
E   ImportError: attempted relative import with no known parent package

Reply:
De bovenstaande melding krijg ik als ik een . toevoeg. Samen met het commando waar je BHV voor nodig hebt, ben ik wel nieuwsgierig hoe ik dit het beste in kan richten?

Eerlijk gezegd snap ik niet hoe ik dit netjes kan regelen, zonder een pip install te doen.